coordinator/src/model/fhir-resources.ts

```diff
@@ -120,11 +121,19 @@ export class PractitionerRole extends Resource {
   readonly resourceType = "PractitionerRole"
   identifier?: Array<Identifier>
   practitioner?: Reference<Practitioner>
-  organization?: Reference<Organization>
+  organization?: PractitionerRoleOrganization
   code?: Array<CodeableConcept>
+  healthcareService?: Array<Reference<HealthcareService>>
   telecom: Array<ContactPoint>
 }
 
+export type PractitionerRoleOrganization = PractitionerRoleOrganizationIdentifier | Reference<Organization>
+
+export interface PractitionerRoleOrganizationIdentifier {
```

Comment:
I don't think this is unique to PractitionerRole. Any Reference can refer to its target either using the reference field or the identifier field. I'd rather we made this generic so we can use it elsewhere (e.g. nominated pharmacy).

Reply:
Also I'm pretty sure identifier shouldn't be an array.

Reply:
I thought so too, but then the example on this page: https://nhsd-confluence.digital.nhs.uk/pages/viewpage.action?pageId=132775978 has an array, and then this page: https://simplifier.net/guide/NHSDigitalSpine/Spine-HealthcareService shows a cardinality of 0...*

I'll change the types to be generic